### PR TITLE
update README 

### PR DESCRIPTION
**Pull request by nan-wang**
- update `README`
- refactor `name` in `setup.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,135 @@
...
+	from jinahub.encoder.clip_image import ClipImageEncoder
+	from jina import Flow, Document
+	import numpy as np
+	
+	f = Flow().add(
+	        uses=jinahub+docker://xgdfljc:v1,
+	        volumes='/your_home_folder/.cache/clip:/root/.cache/clip')
+	
+	def check_emb(resp):
+	    for doc in resp.data.docs:
```
Comment (JoanFM) [inline README.md:46]: I think for this u need to access `resp.docs`, otherwise won't work

File: README.md [hunk 2]
```diff
@@ -4,30 +4,135 @@
...
+	with f:
+		f.post(
+		    on='/foo', 
+		    inputs=Document(np.random.random((224, 224, 3))), 
+		    on_done=check_emb)
+	```
+
+
+
+## Parameters
```
Comment (JoanFM) [inline README.md:121]: I think we need an exhaustive list

File: README.md (hunk 1 above)
Comment (nan-wang) [reply README.md:46]: Nope, it works.

File: README.md (hunk 2 above)
Comment (nan-wang) [reply README.md:121]: hmmm, i think we need to find a way to include docstrings into README automatically.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**